examples/templating/index.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://unpkg.com/tailwindcss@2/dist/tailwind.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="stylesheet" href="Recommend.css" /> 
+
+    <title>Algolia JavaScript Recommend - Templating</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>Algolia Recommend</h1>
+
+      <li>
+        <a href="./createElement/index.html">createElement</a>
+      </li>
+      <li>
+        <a href="./jsx/index.html">JSX</a>
+      </li>
```

Comment:
Applies to the `jsx/index.html` and `createElement/index.html` too.

```suggestion
      <ul>
        <li>
          <a href="./createElement/index.html">createElement</a>
        </li>
        <li>
          <a href="./jsx/index.html">JSX</a>
        </li>
      </ul>
```